Gemfile

```diff
@@ -12,3 +12,5 @@ when /3/
 else
   gem 'mongoid', version
 end
+
+gem 'request_store', '~> 1.3.1' if ENV['USE_REQUEST_STORE']
```

Comment:
I would change `USE_REQUEST_STORE` to `REQUEST_STORE` and set it to the version, so `REQUEST_STORE=~> 1.3.1` in .travis.yml. It's more idiomatic.